### ISSUE TITLE
ml_classiifer api route not found

### ISSUE THREAD

**Issue by MattGrouchy** [pacefactory/deployment-scripts#17]
### Bug

`localhost/api/classifier` is Not found while the ml_classifier is running and can be reached with localhost:7373

**Comment by MattGrouchy** [comment]
@jkingry 
https://github.com/pacefactory/deployment-scripts/blob/c593a349cdedfe9bafcaeef3dbcc08ea18403300/docker-compose.yml#L175
Could this be the reason that `localhost/api/classifier` is not being created?

**Comment by jkingry** [comment]
@MattGrouchy yes, I don't have a way to set the environment variable in the `apigateway` container based on what `--profile ml` argument is set, the seleected `profiles` aren't exposed in another way as far as I could tell
